src/renderer/components/deposit/withdraw/Withdraw.tsx

```diff
@@ -1,65 +1,282 @@
-        })),
-        (feesRd) => <Fees fees={[feesRd]} />
+
+  const oFees: O.Option<WithdrawFees> = useMemo(() => FP.pipe(fees, RD.toOption), [fees])
+
+  const isThorMemoFeeError: boolean = useMemo(() => {
+    if (withdrawPercent <= 0) return false
+
+    return FP.pipe(
+      sequenceTOption(oFees, oRuneBalance),
+      O.fold(
+        // Missing (or loading) fees does not mean we can't sent something. No error then.
+        () => !O.isNone(oFees),
+        ([{ thorMemo }, balance]) => balance.amount().isLessThan(thorMemo.amount())
       )
-    ],
-    [fee]
+    )
+  }, [oFees, oRuneBalance, withdrawPercent])
+
+  const renderThorMemoFeeError = useMemo(() => {
+    const runeBalance = FP.pipe(
+      oRuneBalance,
+      O.getOrElse(() => ZERO_BASE_AMOUNT),
+      baseToAsset
+    )
+
+    return FP.pipe(
+      oFees,
+      O.map(({ thorMemo }) => {
+        const thorFeesAmount: BaseAmount = baseAmount(thorMemo.amount())
+        const msg = intl.formatMessage(
+          { id: 'deposit.withdraw.add.error.thorMemoFeeNotCovered' },
+          {
+            fee: formatAssetAmountCurrency({
+              amount: baseToAsset(thorFeesAmount),
+              asset: AssetRuneNative,
+              trimZeros: true
+            }),
+            balance: formatAssetAmountCurrency({ amount: runeBalance, asset: AssetRuneNative, trimZeros: true })
+          }
+        )
+        // `key`  has to be set for any reason to avoid "Missing "key" prop for element in iterator"
+        return <Styled.FeeErrorLabel key="memo2-fee-error">{msg}</Styled.FeeErrorLabel>
+      }),
+      O.getOrElse(() => <></>)
+    )
+  }, [oRuneBalance, oFees, intl])
+
+  /**
+   * Helper to calculate chain fee
+   * Fee might be calculated into price of asset
+   */
+  const calculateAssetChainFee = useCallback(
+    (fee: BaseAmount): BaseAmount => {
+      // ignore price calculations for assets, which are chain assets
+      if (eqAsset.equals(asset, getChainAsset(asset.chain))) return fee
+      // calculate asset price of fee
+      return getValueOfAsset1InAsset2(fee, assetPoolData, chainAssetPoolData)
+    },
+    [asset, assetPoolData, chainAssetPoolData]
   )
+  /**
+   * Asset chain fee
+   */
+  const oAssetChainFee: O.Option<BaseAmount> = useMemo(
+    () =>
+      FP.pipe(
+        oFees,
+        O.map(({ assetOut }) => calculateAssetChainFee(assetOut))
+      ),
+    [calculateAssetChainFee, oFees]
+  )
+
+  const isAssetChainFeeError = useMemo(() => {
+    if (withdrawPercent <= 0) return false
+
+    return FP.pipe(
+      oAssetChainFee,
+      O.fold(
+        // Missing (or loading) fees does not mean we can't sent something. No error then.
+        () => !O.isNone(oFees),
+        (fee) => assetToBase(withdrawAmounts.assetWithdraw).amount().isLessThan(fee.amount())
+      )
+    )
+  }, [oAssetChainFee, oFees, withdrawAmounts.assetWithdraw, withdrawPercent])
+
+  const renderAssetChainFeeError = useMemo(() => {
+    return FP.pipe(
```

Comment:
same here. maybe 
```ts
if(isAssetChainFeeError) {return null}
```

Reply:
We have a [check in line 351](https://github.com/thorchain/asgardex-electron/pull/662/files/d13ce1e91288667eed01337ff61c1bdf12edb5c3#diff-9523733ca0dcea33995c77fb2a65e28b89a9873ae0e9443005f3180c08fc2f5eR349)